viam-cartographer_test.go

```diff
@@ -336,3 +338,14 @@ func TestSLAMProcess(t *testing.T) {
 
 	testhelper.ClearDirectory(t, dataDir)
 }
+
+// SetupTestGRPCServer sets up and starts a grpc server.
+// It returns the grpc server and the port at which it is served.
+func setupTestGRPCServer(tb testing.TB) (*grpc.Server, int) {
```

Comment:
we had to move this back into the same package in order to not handle the error return value of grpcServer.Serve. see also https://github.com/viamrobotics/viam-cartographer/pull/31/files#r1151106610

the grpc server being set up here is a mock that simulates carto_grpc_server